v2/manager.go

```diff
@@ -30,7 +30,7 @@ import (
 	"syscall"
 	"time"
 
-	"github.com/containerd/cgroups/v2/stats"
+	"github.com/containerd/cgroups/v2/v2/stats"
```

Comment:
This is ugly, but if we move types from v2/v2 to v2, having cgroups v3 (if there is) is a headache. So I'd stay with this simple-yet-ugly naming.

Reply:
Having github.com/containerd/cgroups/v2/cgroup2 (instead of v2/v2) could be an option, by following the filesystem type.

https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html

Reply:
Can we just rename that to “unified” ?

Reply:
The cgroup v1 impl can be named “legacy”

Reply:
`unified` may work, but `legacy` is not future-proof. Would cgroup v2 be considered as legacy eventually?

Reply:
Hmm, but unified.NewManager doesn't look better than cgroups.NewManager.

Reply:
I'm settled to use cgroup2 now.